pkgs/tools/system/btop/default.nix

```diff
@@ -14,14 +17,36 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-uKR1ogQwEoyxyWBiLnW8BsOsYgTpeIpKrKspq0JwYjY=";
   };
 
+  nativeBuildInputs = lib.optionals stdenv.isDarwin [ gcc11 ];
```

Comment:
When a darwin package requires gcc we tend to achieve this by switching out the `stdenv` for a `gccStdenv` e.g. `gcc11Stdenv`. Supplying gcc as an additional input results in a funny environment that has access to both compilers. See usages of `gccStdenv` in `all-packages.nix` for examples.

Reply:
So, in my ignorance in how nixpkgs works, I'm actually seeing in all-packages.nix that stdenv for btop is already being set to gcc11Stdenv. I based the darwin implementation off of what I came up with for a local package I have set up to use btop on my machines, which of course was not setting stdenv to gcc11Stdenv. Thank you for pointing this out, I never would have though to look there. Will be updating the PR to remove that altogether.